Source/1.3/Events/EventWorker.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Empire_Rewritten.Settlements;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Empire_Rewritten.Events
+{
+    public class EventWorker
+    {
+        public EventDef def;
+        public virtual float Chance
+        {
+            get
+            {
+                return 0f;
+            }
+        }
+        public virtual void Event(Empire empire)
```

Comment:
Can this class and this function specifically be abstract? This would force whoever makes an EventWorker to implement this function, which could otherwise remain empty on default (not really a problem, but cleaner?)